packages/portal/spec-renderer-mini/src/components/SpecRendererMini.vue

```diff
@@ -182,21 +231,54 @@ const hasRequiredProps = (items: SpecItemType[]): boolean => {
     }
   }
 
+  .mini-spec-renderer-untagged {
+    margin-left: -12px;
+  }
+
   .is-summary {
     .mini-spec-renderer-section {
+      border: 1px solid var(--grey-200);
+      border-radius: 4px 4px 0 0;
+      padding: 10px 8px 10px 12px;
+
+      .mini-spec-renderer-section-toggle {
+        height: 18px;
+        align-self: end;
+      }
+
       .mini-spec-renderer-section-label {
         font-size: var(--type-lg);
       }
+
+      .mini-spec-renderer-section-description {
+        font-family: monospace;
+        max-width: 65%;
+      }
+    }
+
+    .mini-spec-renderer-untagged {
+      margin-top: 12px;
+      margin-left: unset;
     }
   }
 }
 </style>
 
 <style lang="scss">
-.kong-portal-spec-renderer-mini {
+.kong-spec-renderer-mini {
   // all but the first have top margin
   .k-collapse + .k-collapse {
     margin-top: var(--spacing-md);
   }
+
+  .k-collapse .k-collapse-heading {
+    margin: 0 !important;
```

Comment:
It would be nice to avoid using !important` if possible, can we use any other css specifier?

Reply:
Unfortunately we don't have an alternative currently because the style I am overriding with the !important has itself a !important on it since it's one of our utility classes.. the alternative would be to update KCollapse to not use the utility class for this style but I wasn't sure the need in here warranted the change there.